packages/cadl-ranch/src/cli/cli.ts

```diff
@@ -267,6 +272,7 @@ async function main() {
           generatorName: args.generatorName,
           generatorVersion: args.generatorVersion,
           generatorCommit: args.generatorCommit ?? getCommit(process.cwd()),
+          reportType: args.reportType,
```

Comment:
`reportType` doesn't make a lot of sense to me, it is not the type of the report but more the mode of the emitter you are using. Maybe just `--emitterMode` or `--mode`